src/client/taskmanager_client.cc

```diff
@@ -81,5 +81,75 @@ ::openmldb::base::Status TaskManagerClient::StopJob(const int id, ::openmldb::ta
+    ::openmldb::taskmanager::RunBatchAndShowRequest request;
+    ::openmldb::taskmanager::ShowJobResponse response;
+
+    request.set_sql(sql);
+    request.set_default_db(default_db);
+    // TODO: Set map of config
+
+    bool ok = client_.SendRequest(&::openmldb::taskmanager::TaskManagerServer_Stub::RunBatchAndShow, &request,
+                                  &response, FLAGS_request_timeout_ms, 1);
+
+    if (ok && response.code() == 0) {
+        if (response.has_job()) {
+            job_info.CopyFrom(response.job());
+        }
+        return ::openmldb::base::Status(response.code(), response.msg());
+    } else {
+        return ::openmldb::base::Status(-1, "Fail to request TaskManager server");
+    }
+}
+
+::openmldb::base::Status TaskManagerClient::ImportOnlineData(const std::string sql,
+                                                             const std::map<std::string, std::string> config,
+                                                             const std::string default_db,
+                                                             ::openmldb::taskmanager::JobInfo& job_info) {
+    ::openmldb::taskmanager::ImportOnlineDataRequest request;
+    ::openmldb::taskmanager::ShowJobResponse response;
+
+    request.set_sql(sql);
+    request.set_default_db(default_db);
+
+    bool ok = client_.SendRequest(&::openmldb::taskmanager::TaskManagerServer_Stub::ImportOnlineData, &request,
+                                  &response, FLAGS_request_timeout_ms, 1);
+
+    if (ok && response.code() == 0) {
+        if (response.has_job()) {
+            job_info.CopyFrom(response.job());
+        }
+        return ::openmldb::base::Status(response.code(), response.msg());
+    } else {
+        return ::openmldb::base::Status(-1, "Fail to request TaskManager server");
+    }
+}
+
+::openmldb::base::Status TaskManagerClient::ImportOfflineData(const std::string sql,
+                                                              const std::map<std::string, std::string> config,
+                                                              const std::string default_db,
+                                                              ::openmldb::taskmanager::JobInfo& job_info) {
+    ::openmldb::taskmanager::ImportOfflineDataRequest request;
+    ::openmldb::taskmanager::ShowJobResponse response;
+
+    request.set_sql(sql);
+    request.set_default_db(default_db);
+
+    bool ok = client_.SendRequest(&::openmldb::taskmanager::TaskManagerServer_Stub::ImportOfflineData, &request,
+                                  &response, FLAGS_request_timeout_ms, 1);
+
+    if (ok && response.code() == 0) {
+        if (response.has_job()) {
+            job_info.CopyFrom(response.job());
+        }
+        return ::openmldb::base::Status(response.code(), response.msg());
+    } else {
+        return ::openmldb::base::Status(-1, "Fail to request TaskManager server");
```

Comment:
use `ReturnCode`? e.g. {kError, "Fail to request TaskManager server"}

Reply:
We will refactor to use the error code later.